src/sst/elements/Samba/PageTableWalker.cc

```diff
@@ -308,7 +395,7 @@ void PageTableWalker::recvResp(SST::Event * event)
 
 	insert_way(WID_Add[pw_id], find_victim_way(WID_Add[pw_id], WSR_COUNT[pw_id]), WSR_COUNT[pw_id]);
 
-	long long int addr = WID_Add[pw_id];
```

Comment:
Not required but it would be good to have an address type to make the code more readable. Something like` typedef uint64_t Address_t`